src/guides/wallet.md

```diff
@@ -0,0 +1,164 @@
+## Seting up a Handshake Wallet 
+
+
+`hsd` offers a powerful, modular way to create and manage handshake wallets. In this guide, I'll walk you through the concepts and features you'll need to know about to get started.
+
+### The Basics
+If you're a seasoned bitcoiner, you can probably skim this section or skip straight ahead to the <a href='#examples'> Examples </a>section. But if you're relatively new or just want a refresher, this section will help you understand how wallets actually work.
+
+### Wallets
+In the most basic sense, a handshake wallet is data that enables you to receive and spend HNS, place bids on names and update resource records for your names. `hsd` implements the latest specifications for structuring wallets that are easy to backup, easy to restore, and that work just as well for a novice making their first transactions as for a business with millions of users.
+
+### Keys to the Game
+If you want to transact with HNS, you'll need keys. Each handshake address is associated with a particular key, and wallets are made up of many different keys. Keys consist of both a private key and a public key. The private key is required for spending and is extremely sensitive information, while a public key can be used to receive HNS  and monitor a particular address. If you want to learn more about how this works, read up on Public-Key Cryptography.
+
+### HD vs. Non-HD, you mean like TVs?
+You may have seen references to "HD" wallets and wondered what that means. HD in this context does not mean "high definition," as I assumed it did at first, but rather "hierarchical deterministic." An HD wallet takes a hierarchy of keys in order and makes it so any key in that sequence can be determined by the one before it. This means that if you can produce the first key in the hierarchy, you can then generate a practically unlimited number of subsequent keys. The specification for HD wallets as implemented in `hsd` is defined by [Bitcoin Improvement Proposal (BIP) 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki).
```

Comment:
This needs a better explanation. Specifically, the description of the keys does not illustrate that there is a tree of keys. Each level specifies a different piece of information. The first level specifies that the wallet is adhering to bip44. The second specifies what coin, i.e. 0' for Bitcoin, etc. Perhaps this is too much information but what is here now is a bit too vague and also technically incorrect.

Reply:
Proper handling of this description also sets the table for explaining accounts, which are specified in the third level.